crates/pico_structures/src/schematic.rs

```diff
@@ -96,7 +302,7 @@ impl Schematic {
     fn get_schematic_id_to_internal_id(
         nbt: &Nbt,
         internal_mapping: &InternalMapping,
-    ) -> Result<(Vec<InternalId>, InternalId), SchematicError> {
+    ) -> Result<(Vec<InternalId>, InternalId, Vec<bool>, Vec<u8>), SchematicError> {
```

Comment:
Since this method returns 4 different things, it would be wise to return a struct with 4 fields to improve readability.